test/e2e/network_segmentation.go

```diff
@@ -932,6 +942,19 @@ spec:
 }
 
 func newPrimaryUserDefinedNetworkManifest(name string) string {
+	cidr := `
+    - cidr: 10.20.100.0/16
+`
+	if isIPv6Supported() && isIPv4Supported() {
+		cidr = `
+    - cidr: 10.20.100.0/16
+    - cidr: 2014:100:200::0/60
+`
+	} else if isIPv6Supported() {
+		cidr = `
+    - cidr: 2014:100:200::0/60
+`
+	}
```

Comment:
nit: I would export this to a helper function

Reply:
done